index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Web Development Course</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <!-- CSS only -->
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+        <script src="https://kit.fontawesome.com/f41acb8d77.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="./fonts/cocogoose.css" />
+        <link rel="stylesheet" href="./css/styles.css" />
+        <link rel="stylesheet" href="./css/about.css" />
+        <link rel="stylesheet" href="./css/menu.css" />
+    </head>
+    <body>
+        <main>
+            <nav class="toggle-nav">
+                <button type="button" class="close-menu"><i class="fa fa-times fa-spin" aria-hidden="true"></i></button>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>CodeCamp Campaign</li>
+                </ul>   
+            </nav>
+     <div class="main-container">
+         <div class="header-wrapper">
+            <nav class="upper-nav">
+                <ul>
+                    <li>f</li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </nav>
+            <header>
+                <a href="index.html" class="logo">CodeCamp Logo</a>
+                <span class="toggle-btn">&#9776;</span>
+                <nav class="lower-nav">
+                    <ul>
+                        <li><a href="about.html">About</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li>CodeCamp Campaign</li>
+                    </ul>   
+                </nav>
```

Comment:
- [ ] Could you please make the navbar items in the center following the cross axis, you can achieve that by giving `justify-content: center;`

![links](https://user-images.githubusercontent.com/80895497/158707100-1ccabc6b-3ad1-44ac-83e3-3c7a4a66879a.PNG)